### PR TITLE
Add blog post for Openziti

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes FerretDB/engineering#158.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+These commands will fetch the Docker Compose file and the default environment file required for OpenZiti setup.
+
+Next, modify the docker compose file and add Postgres with a known user/password and FerretDB should connect to the Postgres database using the `FERRETDB_POSTGRES_URI`, as shown below.
+
+```yaml
+postgres-db:
+  image: postgres
+  #ports:
+  #  - 5432:5432
```
Comment (AlekSi) [inline website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md:49]: I guess that should be removed

File: website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+- Ziti Console: Provides a web interface for managing the Ziti network.
+
+FerretDB connects to PostgreSQL and is configured to run within the same Ziti network.
+
+### Initialize Docker Environment
+
+Using the project name 'pg' for Postgres, start the Docker Compose environment:
+
+```sh
+docker compose -p pg up
```
Comment (AlekSi) [inline website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md:87]: Where does that project name come from? Why not just remove `-p pg`?

File: website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+Run `docker ps` to show that Postgres and FerretDB are not exposed (`5432`/`27017`):
+
+### Testing your network connection
+
+To verify the security and functionality of your setup, follow these steps to test network connectivity between the Ziti components and ensure they're correctly configured.
+
+To begin, access the running Ziti Controller container using the following command:
+
+```sh
+docker exec -it pg-ziti-controller-1 bash
```
Comment (AlekSi) [inline website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md:101]: That could be simplified with `docker compose exec`

File: website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+Using the project name 'pg' for Postgres, start the Docker Compose environment:
+
+```sh
+docker compose -p pg up
+```
+
+This command sets up several services, including `ziti-controller`, `ziti-edge-router`, `ziti-console`, `postgres-db`, and `ferretdb`, as specified in the Docker Compose YAML.
+
+Run `docker ps` to show that Postgres and FerretDB are not exposed (`5432`/`27017`):
```
Comment (AlekSi) [inline website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md:92]: `docker compose ps`

File: website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+```
+
+These tests ensure that your Docker network settings allow for proper communication paths between the Ziti Controller and the Ziti Edge Router, ensuring that your FerretDB setup with OpenZiti operates more efficiently and securely.
+
+### Connecting to FerretDB
+
+Once your services are up and running, you can connect to FerretDB using the MongoDB shell (`mongosh`) over the secure network established by OpenZiti.
+
+```sh
+docker run -it --rm --network pg_ziti mongo mongosh "mongodb://postgres:postgres@pg-ferretdb-1:27017/ferretdb?authMechanism=PLAIN"
```
Comment (AlekSi) [inline website/blog/2024-03-18-run-ferretdb-securely-using-openziti.md:154]: I think it is better to add mongo service to Docker Compose configuration to use `docker compose run` and avoid `pg_` prefix everywhere

**Review by ptrfarkas — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**